tools/verify_images.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utility script to verify that all images have alt text"""
+
+import multiprocessing
+import sys
+import glob
+
+# Dictionary to allowlist lines of code that the checker will not error
+# Format: {"file_path": [list_of_line_numbers]}
```

Comment:
I wasn't sure about what was the best granularity for the allowlist, but I ended up choosing this format to allow the content team to work one image at a time, if needed, inside a single file. There a files with a lot of images that maybe can't be fixed at once. However, I think making the allowlist to work with files directly is a good approach too.

Reply:
I think this makes a lot of sense to do line numbers because each individual image may take some time. I like it.